examples/linelast/linelast.yml

```diff
@@ -0,0 +1,35 @@
+main:
+  mode: single_run
+  use_rom: false
+  solver: linelast
+
+mesh:
+  filename: meshes/beam-tri.mesh
+  uniform_refinement: 1
+
+domain-decomposition:
+  type: none
+
+solver:
+  direct_solve: true
+
+discretization:
+  order: 1
+  full-discrete-galerkin: true
```

Comment:
I didn't know you change this option name in the previous PR. We have on-going workflow using this option and you can't just simply change this and break the backward compatibility.

Please change the line 83 of `src/MultiBlockSolver.cpp` back to original input option name.